client/components/Windows/Settings.vue

```diff
@@ -409,6 +415,7 @@ your nickname or expressions defined in custom highlights."
 					<RevealPassword v-slot:default="slotProps">
 						<input
 							id="old_password_input"
+							autocomplete="on"
 							:type="slotProps.isVisible ? 'text' : 'password'"
```

Comment:
while we are at it, can we please give the proper hint for the thing we actually want? https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#:~:text=current-password in that case, and the other password fields should probably be `new-password` so that password manager trigger.

Reply:
Neat, didn't know this existed.